src/dsp_tools/commands/rosetta.py

```diff
@@ -21,7 +21,7 @@ def _update_possibly_existing_repo(rosetta_folder: Path) -> bool:
     is_rosetta_up_to_date = True
     if rosetta_folder.is_dir():
         print(f"Execute 'git pull' in {rosetta_folder}...")
-        completed_process = subprocess.run("git pull", shell=True, cwd=rosetta_folder, check=False)
+        completed_process = subprocess.run("git pull".split(), cwd=rosetta_folder, check=False)
```

Comment:
With the `split()` trick, I avoid `shell=True`, but I also trick https://docs.astral.sh/ruff/rules/start-process-with-partial-path/ away.